### PR TITLE
create PR automation

### PR DESCRIPTION
**Pull request by zreigz**
``` 
plural cr pr-autoamtion create ID --context
```

## Checklist

- [ ] If required, I have updated the Plural documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] I have added a meaningful title and summary to convey the impact of this PR to a user.
- [ ] I have added relevant labels to this PR to help with categorization for release notes.

### CODE REVIEW

**Comment by zreigz** [comment]
@michaeljguarino PTAL

File: cmd/plural/cd_prautomation.go [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+	"os"
+
+	"github.com/pluralsh/plural-cli/pkg/utils"
+	"github.com/samber/lo"
+	"github.com/urfave/cli"
+)
+
+func (p *Plural) cdPrAutomations() cli.Command {
+	return cli.Command{
+		Name:        "pr-automation",
```
Comment (michaeljguarino) [inline cmd/plural/cd_prautomation.go:14]: I think this should be within the `pr` subcommand (I think there's already one).  That way `plural pr create ...` reads a lot more clearly

Comment (zreigz) [reply cmd/plural/cd_prautomation.go]: done

**Comment by zreigz** [comment]
@michaeljguarino I had to migrate it to the new structure, PTAL

**Review by michaeljguarino — APPROVED** [review]
we should also have a way to create by automation name too (don't know if this would require fetching the pr automation first to grab its id though)

**Comment by zreigz** [comment]
> we should also have a way to create by automation name too (don't know if this would require fetching the pr automation first to grab its id though)

I will create an issue for it

**Pull request merged**